docs/ide-usage.md

```diff
@@ -64,8 +64,7 @@ The following plugins are needed:
      Configure for all the languages used in TEAMMATES:
      * Java: `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`.
      * JavaScript: `JavaScript → Code Style → Formatter → Edit → Tab policy → Spaces only`.
-     
-     You can find the Web and XML options in `Eclipse → Preferences → ...`.
+     <br>You can find the Web and XML options in `Eclipse → Preferences → ...`.
```

Comment:
Oops I realised the trailing spaces were to maintain the indentation; my bad 😅 Can you revert the change?